framework/src/modules/pos/endpoint.ts

```diff
@@ -41,10 +41,11 @@ import {
 	StakerData,
 	StakerDataJSON,
 	GetExpectedSharedRewardsRequest,
+	PunishmentLockingPeriods,
 } from './types';
 import { getPunishTime, getWaitTime, isCertificateGenerated, calculateStakeRewards } from './utils';
 import { GenesisDataStore } from './stores/genesis';
-import { EMPTY_KEY, PUNISHMENT_PERIOD } from './constants';
+import { EMPTY_KEY, PUNISHMENT_WINDOW_SELF_STAKING } from './constants';
```

Comment:
Do we really need `PUNISHMENT_WINDOW_SELF_STAKING` instead of the config constant here?